song_match/effect/effects/game_won.py

```diff
@@ -0,0 +1,26 @@
+from cozmo.anim import Triggers
+
+from song_match.effect.effect import Effect
+
+class GameWonEffect(Effect):
+
+    async def play(self, is_player: bool = True) -> None:
+        """Play the correct sequence effect.
+
+        * Play ``GameWonEffect.wav``
+        * Animate Cozmo with :attr:`~cozmo.anim.Triggers.FrustratedByFailureMajor` or
+          :attr:`~cozmo.anim.Triggers.DanceMambo` depending upon ``is_player``.
+        * Flash the cubes green.
+
+        :keyword is_player: Whether the player or Cozmo played the correct sequence.
+        :return: None
+        """
+        animation = Triggers.FrustratedByFailureMajor if is_player else Triggers.DanceMambo
+        if is_player:
+            await self._song_robot.cozmo_lose()
+        else:
+            await self._song_robot.cozmo_win()
+        self._sound.play()
+        await self._note_cubes.flash_lights_green()
```

Comment:
We should have a special cube effect for the game over that's distinct from when you get the notes correct.

Any ideas?

Reply:
Maybe we could just flash the cubes three times with their set colors? Kind of a mixture between the round transition and the flashing green lights